### PR TITLE
Handle tombstones in clusterloader2

### PR DESCRIPTION
**Pull request by wojtek-t**
Fix https://github.com/kubernetes/perf-tests/issues/411

### CODE REVIEW

**Comment by wojtek-t** [comment]
Fixed type

File: clusterloader2/pkg/measurement/common/simple/wait_for_controlled_pods.go [hunk 1]
```diff
@@ -171,6 +171,9 @@ func (w *waitForControlledPodsRunningMeasurement) start(clients *framework.Multi
 		},
 		DeleteFunc: func(obj interface{}) {
 			deleteF := func() {
+				if tombstone, ok := obj.(cache.DeletedFinalStateUnknown); ok {
+					w.handleObject(clients, tombstone.Obj, nil)
```
Comment (krzysied) [inline clusterloader2/pkg/measurement/common/simple/wait_for_controlled_pods.go]: Should it handle `tombstone.Obj` and then `obj`, or is here a `return` missing?

Comment (wojtek-t) [reply clusterloader2/pkg/measurement/common/simple/wait_for_controlled_pods.go]: should return - will fix

**Comment by wojtek-t** [comment]
PTAL

**Pull request merged**